docs/release-notes/2.37.0.md

```diff
@@ -0,0 +1,170 @@
+
+**Experimental Kubernetes Pods Status Dashboard**
+
+*Kubernetes Pods Status dashboard* provides detailed information about the state and performance of your pods, including CPU, Memory, and Network metrics. 
+
+This dashboard can help you quickly pinpoint any issues affecting your pods and ensure they continue to operate smoothly.
+
+**Experimental Kubernetes Volume Dashboard**
+
+*Kubernetes Volumes* dashboard  provides insights into your Kubernetes volumes, including capacity and usage, in real-time. With this dashboard, you can easily monitor the performance and usage of your volumes and take proactive measures to ensure their performance.
+
+To learn more about these new experimental dashboards and how to use them, refer to the [documentation](). 
+
+Here are the steps to create a new folder and move all experimental dashboards to the new folder for quick access and internal use:
+
+!!! note alert alert-primary "Note"
+    You should have at least an *Editor* role to create a new folder and move all experimental dashbaords.
+
+1.	Navigate to the **Main** menu and hover on the {{icon.dashboards}} _Dashboards_ icon.
+2.	Click **New folder**.
+3.	Provide a name for your folder, and then select **Create**.
+4.	Navigate to {{icon.dashboards}} _Dashboards_ from the **Main** menu and click **Browse**.
+5.  Select the dashboard that you want to move and click **Move**.
+6.	On the **Choose Dashboard** dialogue box, from the dropdown under **Folder** option, choose the folder where you want to move the dashboard.
+7.	To apply your changes, select **Save Dashboard**.
+
+
+## New Features
+
+- [PMM-10913](https://jira.percona.com/browse/PMM-10913) - Starting with PMM 2.37.0, you can use an external PostgreSQL server as a data storage for PMM. Thus, even if PMM fails, your data will be stored in an external source, reducing the risk associated with data loss. Furthermore, clustered PostgreSQL instances also enhance performance.
+
+- [PMM-11281](https://jira.percona.com/browse/PMM-11281) - We have written a query for the postgres exporter to deal with lock conflicts in databases that block several transactions, thus impacting performance. With this query, you can find the PID of the blocking session and how many sessions it blocked. The proactive approach enables customers to address non-responsive databases quickly by understanding the underlying causes.
+
+- [PMM-11384](https://jira.percona.com/browse/PMM-11384) - PMM 2.37.0 now supports PSMDB operator version 1.13.
+
+- [PMM-11438](https://jira.percona.com/browse/PMM-11438) - Starting with PMM 2.37.0, PMM administrators can disable the internal PostgreSQL server, which helps to fine-tune the server.
+
+- [PMM-11439](https://jira.percona.com/browse/PMM-11439) - Starting with PMM 2.37.0, a PMM administrator can now view metrics from an external PostgreSQL server.
+
+- [PMM-11675](https://jira.percona.com/browse/PMM-11675) - Telemetry with external PostgreSQL
```

Comment:
let's remove it. it doesn't work fully yet.

Reply:
Removed